criu/image.c

```diff
@@ -226,7 +226,8 @@ int prepare_inventory(InventoryEntry *he)
 	if (get_task_ids(&crt.i))
 		return -1;
 
-	he->has_root_cg_set = true;
+	if (!opts.uid)
```

Comment:
As I already suggested, we need to check opts.unprivileged instead of opts.uid. First, it is guaranteed that a user knows that the unprivileged mode is inuse. Second, it will simplify testing of this feature.

Reply:
Sorry, I realized you're probably looking at the changes patch by patch. All the `opts.uid` issues were fixed in ef9d3a0b6a5a31e3f63a7a1d4d68fb8796c1e26d.